### ISSUE TITLE
Support ES6-style module imports, exports.

### ISSUE THREAD

**Issue by ajvincent** [ajvincent/es7-membrane#33]
Node, CommonJS and browserify are all nice to have, but this is really supposed to be an ECMAScript-friendly module in all respects.  That means import and export statements, hopefully without breaking existing browsers and Node targets...

**Comment by ajvincent** [comment]
Blocked in Firefox by https://bugzilla.mozilla.org/show_bug.cgi?id=1240072 .  

https://bugzilla.mozilla.org/show_bug.cgi?id=1330657 makes this possible via a preference, "dom.moduleScripts.enabled", in FF54.

**Comment by ajvincent** [comment]
https://venturebeat.com/2017/09/05/chrome-61-arrives-with-javascript-modules-and-webusb-support/

**Comment by ajvincent** [comment]
Blocked by https://github.com/karma-runner/karma/issues/2903
